lightning/src/util/events.rs

```diff
@@ -602,6 +620,41 @@ pub enum Event {
 		/// Destination of the HTLC that failed to be processed.
 		failed_next_destination: HTLCDestination,
 	},
+	/// Indicates that a channel featuring anchors outputs is to be closed by broadcasting the local
+	/// commitment transaction. Since commitment transactions have a static feerate pre-agreed upon,
+	/// they may need additional fees to be attached through a child transaction using the popular
+	/// Child-Pays-For-Parent (CPFP) fee bumping technique. This child transaction must include the
+	/// anchor input found within `anchor_descriptor` along with additional inputs to meet the
+	/// target feerate. Once the transaction is constructed, it must be fully signed for and
+	/// broadcasted by the consumer of the event along with the `commitment_tx` enclosed. Note that
+	/// the `commitment_tx` must always be broadcast first, as the child anchor transaction depends
+	/// on it.
+	///
+	/// The consumer should be able to sign for any of the additional inputs included within the
+	/// child anchor transaction. To sign its anchor input, a [`InMemorySigner`] should be
+	/// re-derived through [`KeysManager::derive_channel_keys`] with the help of
+	/// [`AnchorDescriptor::channel_keys_id`] and [`AnchorDescriptor::channel_value_satoshis`].
+	///
+	/// It is possible to receive more than an instance of this event if a valid child anchor
+	/// transaction is never broadcast or is but not with a sufficient fee to be mined. Care should
+	/// be taken by the consumer of the event to ensure any future iterations of the child anchor
+	/// transaction adhere to the Replace-By-Fee (RBF) rules for fee bumps to be accepted into the
+	/// mempool, and eventually the chain.
+	///
+	/// [`InMemorySigner`]: crate::chain::keysinterface::InMemorySigner
+	/// [`KeysManager::derive_channel_keys`]: crate::chain::keysinterface::KeysManager::derive_channel_keys
+	BumpChannelClose {
+		/// The target feerate to use for the child transaction spending the anchor output of the
+		/// channel's commitment transaction to broadcast and bump the fee of.
+		target_feerate_sat_per_1000_weight: u32,
+		/// The channel's commitment transaction to bump the fee of. This transaction should be
+		/// broadcast along with the anchor transaction constructed as a result of consuming this
+		/// event.
+		commitment_tx: Transaction,
```

Comment:
Yeah definitively a good call to have a generic `BumpTransaction` to avoid introducing in the future new events for splicing/dual-funding fee-bumping!